tests/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "scripts": {
-    "cypress:run:chrome": "cypress run --headless --browser chrome",
+    "cypress:run:chrome": "cypress run --env coverage=false --headless --browser chrome",
```

Comment:
Is this enabled by default?
Can we disable coverage for Firefox by default? (by passing the same argument) And override it in .travis.yml?

Reply:
Do you use this arg in the code explicitly?

Reply:
Yes, coverage measurement is enabled by default in the plugin.
Alternatively, you can make 2 commands for each launch. With and without coverage measurement.
I use it as an argument for running through npm.